getting-started/dev/net.md

```diff
@@ -4,28 +4,28 @@
 
 Suppose we wish to run inference on a document using Lucidtech’s invoice model.
 
-```cpp
+```cs
 using Lucidtech.Las;
 
-ApiClient apiClient = new ApiClient("<endpoint>");
-Prediction response = apiClient.Predict(documentPath: "document.pdf", modelName: "invoice|receipt|documentSplit");
-Console.WriteLine(response.ToJsonString(Formatting.Indented));
+Client apiClient = new Client(<credentials>);
```

Comment:
Just 'client' now?

Reply:
If you mean variable name, I don't see the difference, but yes, `ApiClient` class has been removed.